invenio_sip2/api.py

```diff
@@ -19,10 +19,30 @@
 
 from __future__ import absolute_import, print_function
 
+from functools import wraps
+
+from flask import current_app
+from pycountry import languages
+
 from .helpers import MessageTypeFixedField, MessageTypeVariableField
+from .models import SelfcheckLanguage
 from .proxies import current_sip2 as acs_system
 
 
+def preprocess_field_value(func):
+    """Decorator to preprocess field value."""
+
+    @wraps(func)
+    def inner(*args, **kwargs):
+        field = kwargs.get('field')
+        value = kwargs.get('field_value')
+        if value and field.callback:
```

Comment:
What's the behavior if 'callback' property is not define on `field` ? I think it will cause an KeyError exception

Reply:
'callback' property is defined in constructor of the field object.